### ISSUE TITLE
Ressource Booking First Function + UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#14]
## User story
1. As a user
2. I want / need to be able to book one of the available desks returned from the search function
3. So that it is reserved for me and I can use it

## Acceptance criteria
* UI page should be identifyable as a booking page (e.g use an appropriate header)
* UI page should display results returned from the search function in hierarchical manner in a list (e.g using a drop down menu) and apply filters given by preferences (e.g the country, location and room)
* UI page should display properties and names of all available ressources
* UI page should offer a book button
* Pressing the book button should apply the booking in the database and provide the user with some kind of feedback regarding if the booking was successful
* Page should provide a search field where the user can search for a ressource name in case he has a favorite desk and should return the requested ressource if found

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
